src/PluginNodes/Boiler2PluginNodes.cs

```diff
@@ -58,6 +59,11 @@ public void AddOptions(Mono.Options.OptionSet optionSet)
             $"Boiler #2 required maintenance interval in minutes\nDefault: {_maintenanceIntervalMinutes}",

```

Comment:
Change the description to seconds and rename the variable as well

Reply:
changed